client-js/app/scripts/app.ts

```diff
@@ -636,7 +644,7 @@ window.addEventListener('WebComponentsReady', () => {
       throw new Error('no credentials');
     } else {
       try {
-        return mgr.login({
+        await mgr.login({
```

Comment:
This was the bug; we were returning instead of waiting for the login to complete or fail.